pkg/storegateway/bucket_index_reader.go

```diff
@@ -413,9 +413,9 @@ func (r *bucketIndexReader) fetchPostings(ctx context.Context, keys []labels.Lab
 				// Return postings and fill LRU cache.
```

Comment:
nitpick: this comment becomes slightly less relevant now

Reply:
Right, done here: https://github.com/grafana/mimir/pull/3400/commits/8659f2a7f499707c7bfc738380f619ce2f9c8115